tests/FieldAttributesValidationTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Styde\Html\Tests;
+
+use Styde\Html\Facades\Field;
+use Illuminate\Validation\Rules\Exists;
+
+class FieldAttributesValidationTest extends TestCase
+{
+    /** @test */
+    function it_validate_that_the_required_attribute_generates_the_required_rule()
+    {
+        $field = Field::text('name', ['required' => true]);
+
+        $this->assertSame(['required'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_not_generate_the_required_rule_if_the_attribute_is_not_present()
+    {
+        $field = Field::text('name');
+
+        $this->assertNotSame(['required'], $field->getValidationRules());
```

Comment:
I don't see this test as necessary, maybe test the nullable rule is generated when the field is not required. Be more precise.